packages/react-components/react-list-preview/src/hooks/useListSelection.tsx

```diff
@@ -0,0 +1,65 @@
+import { SelectionHookParams, useControllableState, useEventCallback, useSelection } from '@fluentui/react-utilities';
+import * as React from 'react';
+import { ListSelectionState } from './types';
+
+export const defaultListSelectionState: ListSelectionState = {
+  isSelected: () => false,
+  toggleItem: () => undefined,
+  selectItem: () => undefined,
+  deselectItem: () => undefined,
+  clearSelection: () => undefined,
+  toggleAllItems: () => undefined,
+  setSelectedItems: () => undefined,
+  selectedItems: [],
+};
+
+export function useListSelection(options: SelectionHookParams = { selectionMode: 'multiselect' }): ListSelectionState {
+  const { selectionMode, defaultSelectedItems, onSelectionChange } = options;
+
+  const [selectedItems, setSelectedItems] = useControllableState({
+    state: options.selectedItems,
+    defaultState: defaultSelectedItems || [],
```

Comment:
```suggestion
    defaultState: defaultSelectedItems,
```

It's fine to have `defaultSelectedItems` as `undefined` as `useControllableState()` will fallback to `initialState`